eevee/metrics.py

```diff
@@ -1,20 +1,59 @@
-from typing import List
+from typing import Callable, List, Optional
 
+import numpy as np
 from sklearn.metrics import confusion_matrix
 
 from eevee.types import SlotLabel
 
 
-def slot_capture_rate() -> float:
-    ...
+def slot_capture_rate(slots_predicted: List[SlotLabel], expected_slot_value: str) -> float:
+    """
+    """
 
+    if isinstance(slots_predicted, list) and slots_predicted:
+        captured_count = slots_predicted.count(expected_slot_value)
+        return captured_count / len(slots_predicted)
+    else:
+        raise ValueError(f"Expected {slots_predicted = } to be a list of size greater than 0")
```

Comment:
You are raising a ValueError while the error could actually be type mismatch also (based on the if condition).

Reply:
yeah that's why I pointed it out to be :  `a list of`